hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/ScanInfo.java

```diff
@@ -174,8 +174,13 @@ public boolean isNewVersionBehavior() {
    * Used for CP users for customizing max versions, ttl and keepDeletedCells.
    */
   ScanInfo customize(int maxVersions, long ttl, KeepDeletedCells keepDeletedCells) {
```

Comment:
Is this method still referenced? Just remove it?

Reply:
It was used by a few tests: TestCompaction, TestDefaultCompaction, TestMajorCompaction.